CHANGELOG.md

```diff
@@ -1,3 +1,102 @@
+# v1.20.0 (2024-05-13)
+
+## OS Changes
+* Update third party packages ([#3939])
+* Enable file system encryption in 5.15 and 6.1 kernels ([#3906], [#3908])
+* Backport fix for loading SELinux modules ([#3907])
+* Add Fabric Manager support ([#3873])
+* Update host containers ([#3947])
+* Add setting to configure ntp options ([#3852] thanks @domgoodwin)
+* Include swap utilities ([#3829])
+* Update kernels to 6.1.87, 5.15.156, 5.10.215 ([#3934], [#3930])
+
+## Orchestrator Changes
+
+### Kubernetes
+* Add Kubernetes 1.30 variants ([#3859], [#3936])
+* Add container-runtime settings to `aws-k8s-*-nvidia` variants ([#3945])
+* Drop Kubernetes 1.25 Metal and VMware variants ([#3896])
+
+### ECS
+* Update ecs-agent to 1.82.3 ([#3939])
+* Use systemd drop-ins to configure the ECS agent ([#3834])
+
+## Build Changes
+* Update twoliter and the SDK ([#3938], [#3880], [#3885], [#3900])
+* Remove libbzip2 ([#3944])
+* Pessimize Rust builds that require the AWS SDK ([#3892])
+* Reduce variant matrix in CI/CD ([#3863])
+* Document package build tools for go dependencies ([#3882])
+* Update Go lints in CI/CD ([#3882])
+* Out-of-tree build enablement
+  * systemd: use build defaults and kernel parameters for unified cgroups ([#3886], [#3935])
```

Comment:
Need to fix the indent.

Reply:
This is for nested bullets in markdown: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#nested-lists